Beginning a Markdown Style Guide

webinista opened the pull request:
Adds documentation for
- Creating call out boxes (closes #2327)
- Adding attributes
- Where to find Markdown resources.

webinista committed:
Beginning a Markdown Style Guide

Adds documentation for

- Creating call out boxes (closes #2327)
- Adding attributes
- Where to find Markdown resources.

webinista committed:
Beginning a Markdown Style Guide

Adds documentation for

- Creating call out boxes (closes #2327)
- Adding attributes
- Where to find Markdown resources.

webinista commented:
Since much of this are how-tos for the content team, it may be a good idea to have someone on that team read through it and offer feedback.

plusjeff commented:
I'll definitely jump in for review on Tuesday! Excited to have a style guide around this.

webinista committed:
Merge branch 'master' into issue#2414

webinista committed:
Merge branch 'master' into issue#2414

webinista committed:
Merge branch 'master' into issue#2414

webinista committed:
Merge branch 'master' into issue#2414

webinista committed:
Merge branch 'master' into issue#2414

crwallace commented on docs/AddingAttributes.md:
```diff
@@ -0,0 +1,99 @@
+	
+The class or id attribute value: 
+
+- must be **placed on a new line**.
```
Can we move this above both code snippets?

crwallace commented on docs/AddingAttributes.md:
```diff
@@ -0,0 +1,99 @@
+	
+Yes, it is possible to add multiple `class` attribute values, or `class` and `id` attribute values in one go.
+
+	- List item 1
```
Should we group list and list items together in the next section?

crwallace commented on docs/AddACalloutBox.md:
```diff
@@ -0,0 +1,21 @@
+
+Use the syntax found in [Adding Attributes](AddingAttributes.md) to add the `.va-callout` class to the end of the call out list.
+
+The syntax here is very specfic.
```
Specific is misspelled here.

crwallace commented on docs/AddACalloutBox.md:
```diff
@@ -0,0 +1,21 @@
+
+The syntax here is very specfic.
+
+- Answers / definitions should be left-aligned with the question or definition term and begin with `:`.
```
These are a little confusing to me. When we say `There should be a single line break between the question or term and the answer`, do we mean between `: Yes if:` and `    -   You participated in Projects 112 or SHAD testing.` or between `Are you eligible for disability benefits?` and `: Yes if:`. This is only confusing to me because it looks like the line break if after `: Yes if:`, however, `Are you eligible for disability benefits?` is the "question".

Maybe it is helpful to label the parts of this section like:
```
Are you eligible for disability benefits?     // <---- Definition
: Yes if:                                                     // <---- Term

    -   You participated in Projects 112 or SHAD testing.          // <---- Answer
```

Also, I doubt the example above is correct.

webinista committed:
Merge branch 'master' into issue#2414

webinista committed:
Merge branch 'master' into issue#2414

webinista committed:
Clarifying ID and class syntax.

- Correcting spelling of 'specific.'
- Clarifying how to add a class name to a definition list.
- Clarifying the syntax for a definition list.
- Refining, reorganizing list, list items section.

webinista committed:
Clarifying ID and class syntax.

- Correcting spelling of 'specific.'
- Clarifying how to add a class name to a definition list.
- Clarifying the syntax for a definition list.
- Refining, reorganizing list, list items section.

webinista committed:
Merge branch 'master' into issue#2414

webinista committed:
Merge branch 'issue#2414' of github.com:department-of-veterans-affairs/vets-website into issue#2414

webinista committed:
Changing `npm install` to `npm cache clean && npm install`. closes #2498

webinista committed:
Merge branch 'master' into issue#2414

webinista committed:
Merge branch 'issue#2414' of github.com:department-of-veterans-affairs/vets-website into issue#2414

webinista committed:
Merge branch 'master' into issue#2414

webinista committed:
Merge branch 'master' into issue#2414

webinista committed:
Merge branch 'master' into issue#2414

webinista committed:
Merge branch 'master' into issue#2414

webinista committed:
Merge branch 'master' into issue#2414

webinista committed:
Merge branch 'master' into issue#2414

crwallace committed:
Merge branch 'master' into issue#2414

webinista committed:
Merge branch 'master' into issue#2414

webinista committed:
Merge branch 'master' into issue#2414

webinista committed:
Merge branch 'master' into issue#2414

webinista commented:
@crwallace does this PR make sense anymore? If not, feel free to close.

webinista committed:
Merge branch 'master' into issue#2414

plusjeff commented:
This is still relevant, but is from a super old revision. Closing, but not clearing the branch just yet.

@webinista : If you have some time in the week, could you port this over to the new vets-website?
